sogs/filtration.py

```diff
@@ -5,9 +5,15 @@
 
 def should_drop_message_with_body(body):
     """return true if we should drop a message given its body"""
+    body = message_body(body)
```

Comment:
This should come *inside* the if for the file existing.

Reply:
i check outside because if the body is invalid format we should drop it regardless